app/assets/stylesheets/custom.scss

```diff
@@ -217,21 +214,21 @@ ul, menu, dir {
 }
 
 @media only screen and (max-width: 1000px) {
-      ul.slides li {
-        width: 33%; } }
-    @media only screen and (max-width: 767px) {
-      ul.slides li {
-        width: 50%; } }
+    ul.slides li {
+      width: 33%; } }
+  @media only screen and (max-width: 767px) {
+    ul.slides li {
+      width: 50%; } }
     ul.slides li a {
       display: block;
       width: 100%;
       height: 100%;
       position: relative; }
-      ul.slides li a:hover .d-hover {
+    ul.slides li a:hover .d-hover {
         display: block; }
-      ul.slides li a img {
+    ul.slides li a img {
         width: 100%; }
-      ul.slides li a p {
+    ul.slides li a p {
```

Comment:
Line should be indented 0 spaces, but was indented 4 spaces<br>Merge rule `ul.slides li a p` with rule on line 177<br>Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 4